tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -35,16 +35,32 @@ private[scalanative] object LLVM {
   def compile(config: Config, paths: Seq[Path]): Seq[Path] = {
     implicit val _config: Config = config
 
-    // generate .o files for all included source files in parallel
-    paths.par.map { srcPath =>
-      val inpath = srcPath.abs
-      val outpath = inpath + oExt
-      val objPath = Paths.get(outpath)
-      // compile if out of date or no object file
-      if (needsCompiling(srcPath, objPath)) {
-        compileFile(srcPath, objPath)
-      } else objPath
-    }.seq
+    // generate .o files for included source files
+    if (config.targetsMsys || config.targetsCygwin) {
```

Comment:
The problem is that, in order to efficiently diagnose `clang` or `clang++` compile errors, the input and output of the compile session must be examined, by launching `sbt` with the `-debug` argument.  However, the output of multiple parallel compilations are interleaved and unreadable.     In some cases (for example, `sandbox3/run`), the time savings of running in parallel are insignificant and not worth the confused log output.

However, it would be better if we could enable and disable parallel execution by a setting in the `sbt` configuration, but I don't know enough about how to do it that way, so this is an expedient that will permit contributing developers to more easily diagnose problems when resolving the remaining issues with `MSYS2` and `CYGWIN` builds.